rpm/Makefile

```diff
@@ -11,17 +11,23 @@ source:
 ifneq ($(filter epel-6 amzn-%,$(DISTRO)),)
 CURL_VERSION=7.34.0
 else
+ifeq ($(filter sles%,$(DISTRO)),)
 CURL_VERSION=7.52.1
 endif
+endif
 
+ifeq ($(filter sles%,$(DISTRO)),)
```

Comment:
Similar confusion here: Do we mean to use `ifneq` here instead of `ifeq`?

Reply:
Addressed above.